### ISSUE TITLE
Rendering of "o-"

### ISSUE THREAD

**Issue by GarkGarcia** [asciitosvg/asciitosvg#21]
Apparently, the string _o-_ gets rendered as a dot and a line. This is all well and good in most cases, since this characters sequence tends to represent, well, a dot a line. However, this poses a problem for hyphenized words, such as _video-game_.

Shouldn't the rendered take this into account? Perhaps we could make it so that _o-_ gets rendered as _a dot and a line_ when preceded by ` `, `-` or `=`. Otherwise it gets rendered as text.

Would this pose any problems I'm not seeing?

**Comment by GarkGarcia** [comment]
I could work on this, as long as it's a viable change in the current renderer (as long as it doesn't involve redesigning the renderer)

**Comment by dhobsd** [comment]
Yeah, I've found that annoying before, myself. There's a similar issue for `x-`. I think it's a viable change; I'm not sure how big of a deal it is to implement, but I imagine there might be some context you can use. The [code block rendering these](https://github.com/asciitosvg/asciitosvg/blob/master/svg.go#L141-L161) checks hint flags on all points in an object. I think looking at context around the surrounding points is probably not a bad idea.

**Comment by GarkGarcia** [comment]
I doesn't that easy of fix, since the `Point` struct doesn't seem to hold any information about the surrounding point. I also don't think storing this information in the  `Point` struct is the better way to go, since this is irrelevant for most `Point` instance.

Maybe we should reconsider how `Point`'s are added to the `points` field of an `Object` instance. Would omitting points with `.Hint == Dot` or `.Hint == Tick` work? In other words, if we where to not insert those kinds of points to `obj.point`, would the output be rendered the way we intended (as plain text instead of as nodes)?

**Comment by GarkGarcia** [comment]
Also, we could make it so that if an `o-` or `x-` section of the diagram is preceeded with regular text, the corresponding `Point`'s `Hint` field doesn't get set to `Dot` or `Tick`.
